scripts/presubmit-tests.sh

```diff
@@ -129,9 +130,17 @@ function markdown_build_tests() {
   return ${failed}
 }
 
+# Perform yaml build tests if necessary, unless disabled.
+function yaml_build_tests() {
+    (( DISABLE_YAML_LINTING )) && return 0
+    subheader "Linting the yaml files"
+    yamllint "${CHANGED_FILES}"
```

Comment:
It will only "lint" changed files, so a "full linting" will be necessary if we don't want contributor to have to fix the lint the first time a yaml is modified in a repository.